common/persistence/visibility/store/sql/query_converter_mysql.go

```diff
@@ -253,3 +256,32 @@ func (c *mysqlQueryConverter) buildSelectStmt(
 		searchattribute.GetSqlDbColName(searchattribute.RunID),
 	), queryArgs
 }
+
+func (c *mysqlQueryConverter) buildCountStmt(
+	namespaceID namespace.ID,
+	queryString string,
+) (string, []any) {
+	var whereClauses []string
+	var queryArgs []any
+
+	whereClauses = append(
+		whereClauses,
+		fmt.Sprintf("(%s = ?)", searchattribute.GetSqlDbColName(searchattribute.NamespaceID)),
+	)
+	queryArgs = append(queryArgs, namespaceID.String())
+
+	if len(queryString) > 0 {
+		whereClauses = append(whereClauses, fmt.Sprintf("(%s)", queryString))
+	}
+
+	return fmt.Sprintf(
+		`SELECT COUNT(1)
+		FROM executions_visibility ev
+		LEFT JOIN custom_search_attributes
```

Comment:
Is join here must have? PostgreSQL and SQLite were able to get away without join.

Reply:
Yes, in MySQL, we have two tables (executions + custom search attributes), so we need the join.